README.md

```diff
@@ -9,6 +9,8 @@ This is the JavaScript version of [OpenTelemetry](https://opentelemetry.io/), a
 
 > This project is currently in Alpha stage. Its APIs can change at any time and it is not intended to be used in production scenarios!
 
+The minimum supported version of NodeJS is `8.5.0` and the maximum supported version is `12`
```

Comment:
I would put something like this : "**All modern browsers and Node versions from `8.5.0` to `12` are supported.**"

Reply:
Based on the fact that the build is failing on the 8.5.0 container, maybe we should even loosen a little more and say "Modern browsers and Active LTS versions are supported, and it has been tested on the latest version of Node 8."